GenerateReport.ipynb

```diff
@@ -30,6 +30,15 @@
     "lic.tail(10)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stylesheet=\"contribs/css/typography.css\""
```

Comment:
Notice that #6 states `contrib/css/`, not ` contribs/css/`,  but this may be subject for a future PR ;-)

Reply:
I just noticed that you fixed the directory in a later commit. OK